lib/library.go

```diff
@@ -206,10 +206,12 @@ func Logout() *C.char {
 	return makeJSONResponse(err)
 }
 
-//ApproveSignRequest instructs backend to complete sending of a given transaction
+//ApproveSignRequest instructs backend to complete sending of a given transaction,
+// empty values for gas or gasPrice will preserve values for these properties
+// as they were defined on the transaction initialization.
 //export ApproveSignRequest
-func ApproveSignRequest(id, password *C.char) *C.char {
-	result := statusAPI.ApproveSignRequest(C.GoString(id), C.GoString(password))
+func ApproveSignRequest(id, password *C.char, gas, gasPrice C.int) *C.char {
```

Comment:
This will break status-react. That's why I would recommend having `ApproveSignRequest` and `ApproveSignRequestWithArgs` so that we do not need to sync merging this PR with status-react changes.